projects/BFDR.Tests/fixtures/json/BirthRecordFakeNoRace.json

```diff
@@ -9,11 +9,11 @@
   "identifier": {
     "extension": [
       {
-        "url": "http://hl7.org/fhir/us/vr-common-library/StructureDefinition/CertificateNumber",
+        "url": "http://hl7.org/fhir/us/bfdr/StructureDefinition/CertificateNumber",
```

Comment:
vr-common-library appears to be the right ext url https://build.fhir.org/ig/HL7/fhir-bfdr/branches/master/StructureDefinition-Bundle-document-birth-report.html